notify-debouncer-full/src/lib.rs

```diff
@@ -79,7 +79,7 @@ use std::{
 
 use time::now;
 
-pub use cache::{FileIdCache, FileIdMap, NoCache, RecommendedCache};
+pub use cache::{FileIdCache, NoCache, RecommendedCache};
```

Comment:
Here has a break change, but it should be recommend to user to using `RecommendedCache`.

Reply:
```suggestion
pub use cache::{FileIdCache, NoCache, RecommendedCache};

#[cfg(not(target_family = "wasm"))]
pub use file_id_map::FileIdMap;
```